src/chocolatey.tests/infrastructure.app/services/RegistryServiceSpecs.cs

```diff
@@ -110,7 +111,7 @@ public void should_contain_values()
             }
         }
 
-        [WindowsOnly]
+        [Platform(Exclude="Unix,Linux,MacOsX")]

```

Comment:
Should be no changes to this file once you've fixed the WindowsOnly Attribute

Reply:
Done